src/components/screens/AddAssetScreen/AddAssetScreen.tsx

```diff
@@ -86,6 +88,8 @@ const AddAssetScreen: React.FC<AddAssetScreenProps> = () => {
       return;
     }
 
+    analytics.trackAddTokenConfirmed(selectedAsset.assetCode);
```

Comment:
this is unrelated to this PR but I just noticed the function here is called handleAddAssetTrustline although I believe it's being used to add both Classic Trustlines and Custom Tokens. Would you mind doing a quick check and rename the function name so it's not misleading? thanks!